src/components/Board.js

```diff
@@ -5,19 +5,24 @@ import PropTypes from 'prop-types';
 
 
 const generateSquareComponents = (squares, onClickCallback) => {
-  // Complete this for Wave 1
-  // squares is a 2D Array, but 
-  //  you need to return a 1D array
-  //  of square components
-
+    const SquareComponents = squares.flat().map(square => {
+      return (
+        <Square disabled = {square.disabled} onClickCallback={onClickCallback} value={square.value} id = {square.id}></Square>
```

Comment:
Two things: a component doesn't need a closing tag. It's self-closing. Also, let's make this a little more readable:

```suggestion
        <Square 
            disabled = {square.disabled} 
            onClickCallback={onClickCallback} 
            value={square.value} 
            id = {square.id}
        />
```